pkgs/development/tools/analysis/hopper/default.nix

```diff
@@ -12,13 +27,13 @@ stdenv.mkDerivation rec {
 
   sourceRoot = ".";
 
-  ldLibraryPath = with pkgs; stdenv.lib.makeLibraryPath  [
-libbsd.out libffi.out gmpxx.out python27Full.out python27Packages.libxml2 qt5.qtbase zlib  xlibs.libX11.out xorg_sys_opengl.out xlibs.libXrender.out gcc-unwrapped.lib
+  buildInputs = [
+    qtbase
   ];
 
-  nativeBuildInputs = [ pkgs.qt5.wrapQtAppsHook ];
-
-  qtWrapperArgs = [ ''--suffix LD_LIBRARY_PATH : ${ldLibraryPath}'' ];
+  qtWrapperArgs = [
+    ''--suffix LD_LIBRARY_PATH : ${ldLibraryPath}''
```

Comment:
This is not correct use of `qtWrapperArgs`, it will break when _structuredAttrs are enabled. Also it is not a good idea to use `LD_LIBRARY_PATH` with programs that execute child processes (as I suspect debugging tool will), since it will apply to them to. I suggest adding the dependencies using patchelf, like the interpreter is, or using `autopatchELFHook`.

Reply:
I was thinking about refactoring this derivation in a second pull request. For now I would just like to get this working again. As you can see from the change, the original derivation did set `LD_LIBRARY_PATH` already.

Reply:
You are already doing refactoring so it might make sense to fix what is broken while at it.

Reply:
done

Reply:
done

Reply:
I think you still need to add the libraries you want `autoPatchelfHook` to include to `buildInputs`, see https://nixos.wiki/wiki/Packaging/Binaries for more info.

Reply:
Oh, I see, most of them do not appear to be needed.